src/utils/objects/getDiffAsPartialObject.ts

```diff
@@ -0,0 +1,62 @@
+import { isEmpty } from './isEmpty'
+import { setInPath } from './setInPath'
+
+const _traverseAndCompare = <T extends Record<string, unknown>>(
+  a: unknown,
+  b: unknown,
+  path: Array<string> = [],
+  keysToIgnore: Array<string> = [],
```

Comment:
for example, we might want to ignore the 'uuid' or 'rowId' as it is not relevant for us (in this specific case)

Reply:
Seems to have problem with arrays, would keep current version
let me know